src/data_loading.jl

```diff
@@ -0,0 +1,62 @@
+abstract type Dataset end
+
+struct Metadata <: Dataset end
+struct TaxonomicProfiles <: Dataset end
+struct UnirefProfiles <: Dataset end
+struct KOProfiles <: Dataset end
+struct ECProfiles <: Dataset end
+struct MetabolicProfiles <: Dataset end
+struct Neuroimaging <: Dataset end
+
+load(ds::Dataset; kwargs...) = MethodError("load has not been implemented for $(typeof(ds))")
+
+load(::Metadata) = CSV.read(datafiles("exports", "timepoint_metadata.csv"), DataFrame)
```

Comment:
It's just a suggestion: I know DataFrames is very smart, but I like to have some control over those things, so I would pass the explicit column types here:

```
    delim = ',',
    types = [
        Int64,      #subject
        Int64,      #timepoint
        Float64,    #ageMonths
        String,     #race
        String,     #maternalEd
        Dates.Date, #assessmentDate
        Dates.Date, #scanDate
        Float64,    #cogScore
        String,     #omni
        String,     #etoh
        Bool        #has_segmentation
        ]
```

btw, this requires `using Dates`